### ISSUE TITLE
Revising: Mixing of Control and Data Flow & Pausing

### ISSUE THREAD

**Issue by drinkcoffee** [CrosschainRiskFramework/CrosschainRiskFramework.github.io#15]
The "Mixing of Control and Data Flow" should be before the "Pausing" section, as the Pausing section uses the concepts of Control Flow and Data Flow. The "Mixing of Control and Data Flow" should be enhanced to have a diagram, to help more clearly explain what is meant by Control Flow and Data Flow.

**Comment by drinkcoffee** [comment]
This issue is being resolved in this PR: https://github.com/CrosschainRiskFramework/CrosschainRiskFramework.github.io/pull/16

**Comment by drinkcoffee** [comment]
In my PR I have:
* I have provided a real world example of issues when the proposed advice has not been taken. I think having real world examples will help people to understand the importance of a particular idea.
* For the examples, I linked to the analysis by rekt.news . I feel that this is a reliable source of detailed analysis that is likely to be around for some years. 
* When referring to "hacks", I have only called them "issues". I feel that this is less inflammatory.
